From 2b145acc5aaaa51d28c1bc7a9c3d235cdba55d5a Mon Sep 17 00:00:00 2001
From: Dimitri John Ledkov <dimitri.ledkov@surgut.co.uk>
Date: Wed, 7 Oct 2026 10:11:27 +0100
Subject: [PATCH] tools: do not escape build location

Sync default gems from within the repository checkout instead of
../../ locations.
---
 tool/sync_default_gems.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tool/sync_default_gems.rb b/tool/sync_default_gems.rb
index 6945c6cdce..15c8ddd380 100755
--- a/tool/sync_default_gems.rb
+++ b/tool/sync_default_gems.rb
@@ -384,7 +384,7 @@ def sync_default_gems(gem)
     config = REPOSITORIES[gem]
     puts "Sync #{config.upstream}"
 
-    upstream = File.join("..", "..", config.upstream)
+    upstream = File.join("upstream", config.upstream)
 
     config.mappings.each do |src, dst|
       rm_rf(dst)
-- 
2.53.0

